i18n/en.json

```diff
@@ -17,5 +17,6 @@
     "image-retrieval-failed": "Image retrieval failed: $1",
     "documentation": "Documentation",
     "version": "Version $1",
-    "report-issue": "Report an issue"
+    "report-issue": "Report an issue",
+    "langs-param-error": "This language is NOT supported by the OCR."
```

Comment:
We'll want to indicate _which_ languages are invalid (there could be multiple), so something like `The following languages are not supported by the OCR engine: $1`. $1 would be a comma-separated list of the invalid languages. You can use Intuition's [`listToText()`](https://github.com/Krinkle/intuition/blob/39cc5fbb3ced1d1bff166bbe3434b5f1953bdd06/src/Intuition.php#L1416) to make that localized (because not every language uses commas). I think you'll have to inject the Intuition service to get access to that method.

Reply:
Thank you for the pointers!! Done.